tests/test_pc.py

```diff
@@ -29,7 +34,7 @@ def getNewForm(self, pc, u0, test):
         return F, bcs
 
 
-def rd(butcher_tableau):
+def rd(scheme, Lhigh=None, **kwargs):
```

Comment:
A comment for the reader (even though it's a test) that when `Lhigh` is nontrivial and you have a Galerkin collocation scheme, this approximates the Jacobian with a lower-order quadrature rule before applying Rana?